sound/soc/sof/intel/pci-apl.c

```diff
@@ -33,6 +33,9 @@ static const struct sof_dev_desc bxt_desc = {
 		[SOF_IPC] = "intel/sof",
 		[SOF_INTEL_IPC4] = "intel/avs/apl",
 	},
+	.default_lib_path = {
+		[SOF_INTEL_IPC4] = "intel/sof-lib/apl",
```

Comment:
we use intel/sof-ipc4/apl for the base firmware, should there be a mention of ipc4 for the libs as well?

If there is ever support for libraries for IPC3 the suggested naming would become ambiguous.

intel/sof-ipc4-lib ?

Reply:
Actually it is `intel/avs/apl`, only MTL have `intel/sof-ipc4/mtl`
Should we have `avs-lib` and `sof-ipc4-lib` depending on where the IPC4 default path is pointing?

Reply:
probably yes, that would be clearer